Settings.jsx

```diff
@@ -227,7 +229,7 @@ module.exports = class Settings extends React.Component {
           Highlight Cute Words
         </SwitchItem>
         <SwitchItem
-          note="Do you want to get notifications in guilds you're lurking?"
+          note="Do you want to get notifications in guilds you are lurking in? (Lurked servers are ones where you can't chat)"
```

Comment:
Lurked is what this used to be called in the UI, it looks like the current terminology Discord uses is "preview mode". This should be updated to that as it's more accurate than "ones where you can't chat".

Reply:
Thanks for pointing this out, I didn't know about a lurking mode to begin with so it was really confusing. Renamed to previewing